pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -886,18 +881,6 @@ func (vs *VSphere) AttachDisk(vmDiskPath string, storagePolicyName string, nodeN
 	return diskUUID, err
 }
 
-func (vs *VSphere) retry(nodeName k8stypes.NodeName, err error) (bool, error) {
```

Comment:
retry() function is not needed since the caller still handle the same condition check, removing this function will make code easier to understand